docs/Netplay.md

```diff
@@ -25,7 +25,7 @@ If you will be acting as the host...
 - Either go to your router settings and open the port 55435 for both TCP and UDP **or** change the TCP/UDP Port setting in RetroArch to one that's already open
 - Select a number for Delay Frames. If you are experiencing a very low fps, try increasing this number.
 - Pick a Nickname
-- When launching a ROM to use netplay you'll open up the [runcommand menu](https://github.com/retropie/retropie-setup/wiki/runcommand) by pressing js0 (or any key on your keyboard) as your ROM loads and then you'll select launch netplay
+- When launching a ROM to use netplay you'll open up the [runcommand menu](runcommand) by pressing js0 (or any key on your keyboard) as your ROM loads and then you'll select launch netplay
```

Comment:
`[runcommand menu](runcommand)` should be `[Runcommand launch menu](Runcommand##runcommand-launch-menu)`.